deploy/acm-policies/50-GENERATED-rosa-ingress-certificate-policies.Policy.yaml

```diff
@@ -0,0 +1,119 @@
+                            namespace: openshift-acm-policies
+                        spec:
+                            disabled: false
+                            policy-templates:
+                                - objectDefinition:
+                                    apiVersion: policy.open-cluster-management.io/v1
+                                    kind: ConfigurationPolicy
+                                    metadata:
+                                        name: rosa-ingress-certificate
+                                    spec:
+                                        evaluationInterval:
+                                            compliant: 2h
+                                            noncompliant: 45s
+                                        object-templates:
+                                            - complianceType: musthave
+                                              metadataComplianceType: musthave
+                                              objectDefinition:
+                                                apiVersion: v1
+                                                data:
+                                                    tls.crt: '{{hub fromSecret "openshift-acm-policies" .ManagedClusterName "tls.crt" hub}}'
+                                                    tls.key: '{{hub fromSecret "openshift-acm-policies" .ManagedClusterName "tls.key" hub}}'
+                                                kind: Secret
+                                                metadata:
+                                                    name: ingress-certificate-secret
+                                                    namespace: openshift-ingress
+                                        pruneObjectBehavior: DeleteIfCreated
+                                        remediationAction: enforce
+                                        severity: low
+                                - extraDependencies:
+                                    - apiVersion: policy.open-cluster-management.io/v1
+                                      compliance: Compliant
+                                      kind: ConfigurationPolicy
+                                      name: rosa-ingress-certificate
+                                      namespace: ""
+                                  objectDefinition:
+                                    apiVersion: policy.open-cluster-management.io/v1
+                                    kind: ConfigurationPolicy
+                                    metadata:
+                                        name: rosa-ingress-replace-default-cert
+                                    spec:
+                                        object-templates:
+                                            - complianceType: musthave
+                                              objectDefinition:
+                                                apiVersion: operator.openshift.io/v1
```

Comment:
Will this replace the default IngressController? Is there a way to patch instead?

Reply:
the `musthave` type will perform a patch of the objectdefinition if it exists.